pkg/lib/cockpit-components-install-dialog.jsx

```diff
@@ -207,3 +218,56 @@
     check_missing();
     return prom;
 }
+
+function install_dialog_immutable(pkg, options) {
+    return new Promise((resolve, reject) => {
+        const missing_name = <strong>{pkg.join(", ")}</strong>;
+
+        const body = {
+            id: "dialog",
+            title: options.immutable_title || _("Add software"),
+            body: (
+                <div className="scroll">
+                    <p>{ format_to_fragments(options.immutable_text || _("$0 needs to be added to the OS image."),
+                                             missing_name) }</p>
+                </div>
+            ),
+        };
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18240-20230130-105647-26c41b66-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)